tools/reviewer/reviewer.proto

```diff
@@ -37,31 +37,13 @@ message ReviewerConfig {
   // Display name for Reviewer instance. Should be same as
   // ReviewerRegistryConfig.display_name
   string display_name = 1;
-  FirebaseConfig firebase_config = 2;
   // Repos in this Reviewer instance
   repeated Repo repo = 3;
   repeated User user = 4;
   repeated Project project = 5;
   int32 total_crystal = 6;
 }
 
-/* Firebase config. The only field we probably need is project_id, but we're not
- * sure of that. */
-message FirebaseConfig {
```

Comment:
I suppose current version of webapp, which is based on it will stop working after that PR. So I'll temporally deploy old version with hard-coded config

Reply:
Yes, it's a breaking change. If it's easier, let's break Reviewer and fix it, no need to push old version..

Reply:
I'm already deployed old version. It doesn't remove any functionality. When #437 is done, I'll deploy it instead

Reply:
Ok